src/vm/arm/ehhelpers.S

```diff
@@ -99,6 +99,7 @@ OFFSET_OF_FRAME=(4 + SIZEOF__GSCookie)
         NESTED_ENTRY CallEHFunclet, _TEXT, NoHandler
 
         PROLOG_PUSH  "{r4-r11, lr}"
+        PROLOG_STACK_SAVE r7
         alloc_stack  4
 
```

Comment:
It is strange that the `.pad` in the alloc_stack doesn't work. I wonder - should the .pad actually precede the sub sp, sp, \Size in the macro? Could you please try if that change alone fixes the problem? Looking at https://sourceware.org/binutils/docs/as/ARM-Unwinding-Tutorial.html, that tutorial uses first the `.pad` and then the `sub sp, sp, #nn` while our macro does the opposite.